### ISSUE TITLE
Page title does not reflect app bar title on docs until reload

### ISSUE THREAD

**Issue by ChrisBrownie55** [prateekbh/preact-material-components#1163]
While on the docs, when you reload the page, the page title reflects what page you are on. When you switch pages, say from _"Button"_ to _"Chips"_, the page title stays the same, it reads _"Button"_.

The page title should probably reflect the current page (I don't know if that will improve SEO but it sounds like it might).

**Comment by cromefire** [comment]
Yes known bug we are currently working on 2.0 so this is not the top priority, but I'll leave this as a reminder
